### PR TITLE
[MIRROR] General taur visuals refactor and other things.

### PR DESCRIPTION
**Pull request by SkyratBot**
Original PR: https://github.com/Citadel-Station-13/Citadel-Station-13/pull/11874
--------------------
## About The Pull Request
Taur assets should now prevent standard legs from being displayed on the owner, amongst other visual improvements partly ported from virgo.
Refactored generate_female_clothing() very slightly, was thinking of using filters at first but render targets require mapping in and icon filters are not multidirectional.
Refactored `handle_mutant_bodyparts` to use a static list instead of a getter proc for layertext and add fewer empty overlays.

long story short, I wanted to delete some accessories but it went out of hand.

## Why It's Good For The Game
tbh, it'd have been better if there were no such cosmetic preferences to start with.

## Changelog
:cl:
refactor: Refactored some cosmetic bodyparts procs.
refactor: Refactored some taur stuff.
imageadd: See above.
imagedel: See above.
/:cl:

### CODE REVIEW

File: code/modules/client/preferences_savefile.dm [hunk 1]
```diff
@@ -181,6 +181,15 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 		if(lickable)
 			ENABLE_BITFIELD(vore_flags,LICKABLE)
 
+	if(current_version < 29)
```
Comment (Ghommie) [inline code/modules/client/preferences_savefile.dm:184]: oh, yea, you need to bump this and the max_savefile_version define to 30 cuz of Putnam's vorey refactor
